src/main/java/org/fundacionjala/coding/sortdescending/Kata.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.sortdescending;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Kata about Sort the inner content of a sentence.
+ */
+public final class Kata {
+    /**
+     * Empty constructor.
+     */
+    private Kata() {
+    }
+
+    /**
+     * @param words Input Word
+     * @return Sort the inner content of the word
+     */
+    public static String sortTheInnerContent(final String words) {
+        String[] origWords = words.split(" ");
+        for (int i = 0; i < origWords.length; i++) {
+            if (origWords[i].length() > 2) {
+                origWords[i] = descendingSort(origWords[i]);
+            }
+        }
+        return String.join(" ", origWords);
+    }
+
+    /**
+     * @param word Input Separated word
+     * @return Sort descending
+     */
+    private static String descendingSort(final String word) {
+        char[] w = word.toCharArray();
+        List<Character> list = new ArrayList<Character>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)